templates/jetty-default.erb

```diff
@@ -25,7 +25,7 @@ JETTY_PORT=8983
 #JETTY_ARGS=
 
 # Extra options to pass to the JVM         
-JAVA_OPTIONS="-Xss1280k -Dsolr.solr.home=/usr/share/solr -Dsolr.data.dir=/var/lib/solr $JAVA_OPTIONS"
```

Comment:
Why remove the $JAVA_OPTIONS variable and hard code the -Djava.awt.headless and -Djava.library.path=/usr/lib values ? 
where is initial value of JAVA_OPTIONS set ?

Reply:
the initial value of JAVA_OPTIONS is set in the init script. (/etc/init.d/jetty)

Reply:
@MWilkinson There. Fixed. EVen though it's confusing to have more than one of the same parameter on the comandline.